ONSdigital/babbage
# Fix - Remove unused templates

### What
Removed all unused handlebars templates from babbage. 
I've gone through each handlebars template under `src/main/web/templates/handlebars` and searched for references to it in any of the code base. These files I could not find references to.

One comment was removed as that was the only referenced to `ns-logo`

Tidied up `main.handlebars` a bit so the templates are ordered numerically.

### How to review
1. Check that you cannot find any reference to the files in babbage.
1. Check that the site still works - this is the hardest bit. I've tried to visit all pages and nothing has crashed.

### Who can review
Anyone but me
